cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -228,20 +173,20 @@ Slice::CsGenerator::typeToString(const TypePtr& type, const string& package, boo
         "float",
         "double",
         "string",
-        "Ice.Object", // not used anymore
-        "Ice.ObjectPrx?",
-        "Ice.Value?"};
+        "global::Ice.Object", // not used anymore
```

Comment:
I thought we decided to not use `global::` for Ice.

Reply:
Oh okay, I removed the `global::` prefixes.